Gemfile

```
@@ -9,6 +9,8 @@ ruby '3.0.3'
 gem 'rails', '~> 6.1'
 # Use postgresql as the database for Active Record
 gem 'pg', '>= 0.18', '< 2.0'
+# Use MySQL as the database for Active Record
+gem "mysql2", "~> 0.5"
```

Can you replace `"` into `'` for fixing rubocop error?

```suggestion
gem 'mysql2', '~> 0.5'
```